selection/database_connector.py

```diff
@@ -7,6 +8,12 @@ def __init__(self, db_name, autocommit=False):
         self.autocommit = autocommit
         logging.debug("Database connector created: {}".format(db_name))
 
+        # This does not reflect the number of unique simulated indexes but
+        # the number of simulate_index calls
+        self.simulated_indexes = 0
+        self.cost_estimation_duration = 0
```

Comment:
I would prefer storing also the number of cost estimations here, although they can be calculated by `CostEstimation.cost_requests - CostEstimation.cache_hits`.